pages/styles/podcasts.css

```diff
@@ -0,0 +1,15 @@
+.content {
+  align-items: center;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  overflow: hidden; /* prevent devs from bloating content */
+  margin: 1.5rem;
+  height: calc(100% - 3rem); /* 1.5rem margin top AND bottom hence 3rem */
+  width: calc(50% - 3rem); /* 1.5rem margin left AND right hence 3rem */
+}
+
+img {
```

Comment:
this will affect every img component on the page, and - if the app wasn't automatically code-split per route - it would affect the whole app. Nowadays you never want to scope css to an element except for sound, global defaults. I recommend changing this to simply `.image` and passing it as a class name like `className={styles.image}` for the images on the page.